lib/genKeys.js

```diff
@@ -1,16 +1,11 @@
 'use strict'
 
-function genKeys (obj, set) {
+function genKeys (obj) {
   return Object.keys(obj)
-    .filter(notInSet, set)
     .filter(noObjects, obj)
     .map(keyIterator, obj)
 }
 
-function notInSet (key) {
-  return !this || this.has(key)
-}
-
```

Comment:
If you follow the logic + coverage reports set is never used. None of that logic actually has a call to match. I think its holdever from the orginal implementation of sets. No path touches it though.